content/docs/components/central-dash/registration-flow.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Registration Flow"
+description = "The view that allows you to set up your first namespace in the cluster"
+weight = 10
++++
+
+Your Kubeflow deployment comes deployed with Kubeflow Central Dashboard. 
+
+This view will show when the cluster is:
+
+- When in `multi-user` isolation mode:
+  - Your user has no profile (workgroup / namespace) role-bindings in which you're `admin` (Owner)
+- When in `single-user` isolation mode:
+  - The entire cluster has no profile (workgroup / namespace) role-bindings
+
+This guide shows you what this view looks like and how the flow appears.
+
+## Quick guide
+Summary of steps:
+
+1. Follow the [Kubeflow getting-started guide](/docs/started/getting-started/) 
+  to set up your Kubeflow deployment and open the Kubeflow UI.
+1. If the registration flow view shows up (because the conditions above are satisfied):
+   - Then the view will walk you through creating the first workgroup binding for your user or cluster (depending on `isolation-mode`)
+
+## Visual
+![Kubeflow registration flow](/docs/images/registration-flow.gif)
+
+#### Transcription
```

Comment:
Rather than a description of what happens in the gif, we should give step-by-step instructions of what the user needs to do. For example, tell them to click **Start Setup**, then tell them to do whatever they need to do next, and so on.

I'd recommend that you add the steps in the numbered list that you've already started above the gif, then have the gif as the last item on the page. (Remove the transcription.)

Reasoning: The gif is hard to follow, as it moves fast and the person viewing it needs to watch it multiple times to understand what'g going on. We should therefore offer it only as an additional helpful tool, rather than the main form of instruction.

Reply:
I see, lmk if this update is better